### PR TITLE
MST-232 Add the instructions on how to get auth_tokens to make API calls documented

### PR DESCRIPTION
**Pull request by schenedx**
## Description
As part of improving the documentation system edX sends to our partnering institution, we want to enhance the api docs web page on registrar to include enough instructions for partners to get started.
See screenshot of this change:

registrardocs

@edx/masters-devs-cosmonauts Please review

### CODE REVIEW

File: registrar/urls.py [hunk 1]
```diff
@@ -36,13 +36,79 @@
...
+            "https://courses.edx.org/api-admin/status which will display your client_id "
+            "and client_secret. "
+            "<br/><br/>"
+            "<b>JWT Token Request</b>"
+            "<br/><br/>"
+            "Once you have your client_id and client_secret, you can make a POST request "
+            "to https://api.edx.org/oauth2/access_token/  which will return a JSON "
+            "dictionary containing the token."
+            "<br/><br/>"
+            "<b>Sample Request & Response</b>"
```
Comment (kdmccormick) [inline registrar/urls.py]: HTML has a `<pre>` tag for wrapping pre-formatted code blocks like this (like ``` in Markdown). Maybe that would improve the formatting here.

**Review by kdmccormick — COMMENTED** [review]
Nice work! Thanks for doing this.

Could we move the HTML to something like `api-description.html`, and then load it via Python with `open()`? It would give us syntax highlighting and make review & modification easier.

**Comment by schenedx** [comment]
@kdmccormick Took in your feedback. Please review again.

**Comment by kdmccormick** [comment]
@schenedx Can you post a new screenshot? My work laptop is out of service currently...

**Comment by schenedx** [comment]
@kdmccormick Here you go:

registrar-docs-feedback

**Review by kdmccormick — APPROVED** [review]
:ship:

**Pull request merged**